website/blog/2020-08-06-presto-in-ea.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Building a high-performance platform on AWS to support real-time gaming services using Presto and Alluxio
+author: Teng Wang
+authorURL: https://www.linkedin.com/in/teng-wang-9b8732194/
+---
+
+**Authors:** Teng Wang, Du Li, Yu Jin and Sundeep Narravula
+
+Electronic Arts (EA) is a leading company in the gaming industry, providing over a thousand games to serve billions of users worldwide.
```

Comment:
providing dozens of games to serve billions of users worldwide every year.

Reply:
@SerenaW1991 can you create a separate PR on the minor fix? this pr has been merged.